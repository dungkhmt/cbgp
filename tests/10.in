20 20
20 36
20 11
11 2
2 3
3 9
9 12
12 8
8 20
5 13
13 4
4 16
16 15
15 7
7 5
6 19
19 1
1 14
14 18
18 6
18 7
6 1
17 10
15 9
10 14
12 15
16 9
13 19
5 8
20 2
5 6
17 14
16 3
12 7
10 18
4 2
20 13
4 1
1
0 0 20 2 21 3 22 0 11 3 8 2 18 10 16 6 22 4 14 14 17 2 18 8 13 1 12 12 19 9 28 0 14 13 17 11 9 1 18 4 
