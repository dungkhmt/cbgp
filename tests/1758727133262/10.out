(19, 10), (10, 1), (1, 2), (2, 8), (8, 11), (11, 7), (7, 19), (4, 12), (12, 3), (3, 15), (15, 14), (14, 6), (6, 4), (5, 18), (18, 0), (0, 13), (13, 17), (17, 5), (17, 6), (5, 0), (16, 9), (14, 8), (9, 13), (11, 14), (15, 8), (12, 18), (4, 7), (19, 1), (4, 5), (16, 13), (15, 2), (11, 6), (9, 17), (3, 1), (19, 12), (3, 0), 
numberIntersection=0 
minDistance=5.0 
minAngle=0.46364760900080615 
sumAngle=21.213878904768812 
sumDistance=17413.629774939902 
minEdgeLength=5.385164807134504 
sumEdgeLength=624.381591349801 

0.0000000000 5.0000000000 0.4636476090 21.2138789048 17413.6297749399 5.3851648071 624.3815913498

0: (5, 0),
1: (36, 5),
2: (42, 7),
3: (30, 0),
4: (17, 13),
5: (11, 11),
6: (23, 34),
7: (29, 20),
8: (42, 19),
9: (20, 74),
10: (30, 7),
11: (32, 25),
12: (21, 8),
13: (0, 39),
14: (50, 42),
15: (56, 0),
16: (0, 61),
17: (18, 36),
18: (14, 5),
19: (33, 16),

Time taken: 7.961 s
5 0 36 5 42 7 30 0 17 13 11 11 23 34 29 20 42 19 20 74 30 7 32 25 21 8 0 39 50 42 56 0 0 61 18 36 14 5 33 16 
0: (5, 0), 1: (36, 5), 2: (42, 7), 3: (30, 0), 4: (17, 13), 5: (11, 11), 6: (23, 34), 7: (29, 20), 8: (42, 19), 9: (20, 74), 10: (30, 7), 11: (32, 25), 12: (21, 8), 13: (0, 39), 14: (50, 42), 15: (56, 0), 16: (0, 61), 17: (18, 36), 18: (14, 5), 19: (33, 16), 
numberIntersection=0 minAngle=0.46364760900080615 sumAngle=21.213878904769754 minEdgeLength=5.385164807134504 sumEdgeLength=624.3815913498017 minDistance=5.0 sumDistance=17413.62977493985 
